packages/scripts/src/admin-proposals/transferTokensFromGovernor.js

```diff
@@ -35,11 +34,17 @@ async function run() {
   const web3Providers = { 1: getWeb3ByChainId(1) }; // netID => Web3
   const web3 = web3Providers[1];
   const tokenAddress = await _getContractAddressByName("VotingToken", 1);
+  const governorAddress = await _getContractAddressByName("Governor", 1);
   console.log("tokenAddress:", tokenAddress);
   const uma = new web3.eth.Contract(ExpandedERC20.abi, tokenAddress);
 
   const gasEstimator = await setupGasEstimator();
 
+  var originalBalance = await uma.methods.balanceOf(recipient).call();
```

Comment:
Nit: Might be better to have this within the if block, since this is misleading to be titled "original balances" when running verification step